api/src/accounts.py

```diff
@@ -441,6 +442,19 @@ def confirm_email():
         {'_id': ObjectId(user['_id'])},
         {'$set': {'confirm_secret': '', 'email_confirmed': True}}
     )
+
+    # BAD TO HARDCODE, CHANGE LATER
+    new_reg = {
+        "event": "Fall 2021", # update 
+        "apply_at": datetime.datetime.utcnow(),
```

Comment:
datetime.datetime.utcnow() gives Greenwich time(UK), could use datetime.now() to get locale time or use 

from datetime import datetime
from pytz import timezone
tz = timezone('EST')
datetime.now(tz) 

to ensure we get eastern time zone

Reply:
Changed, but unfortunately Mongo always stores dates as UTC anyway :/